### PR TITLE
Getresponse new actions and sources

### PR DESCRIPTION
**Pull request by jcortes**
# Actions
- [x] Create Contact | Creates a contact. [See the docs here](https://apireference.getresponse.com/?_ga=2.47520738.499257728.1666974700-2116668472.1666974700&_gl=1*1f3h009*_ga*MjExNjY2ODQ3Mi4xNjY2OTc0NzAw*_ga_EQ6LD9QEJB*MTY2Njk3NzM0Ny4yLjEuMTY2Njk3ODQ3OS4zNi4wLjA.#operation/createContact)
- [x] Create or Update Contact | Creates or updates a contact if already exists. [See the docs here](https://apireference.getresponse.com/?_ga=2.47520738.499257728.1666974700-2116668472.1666974700&_gl=1*1f3h009*_ga*MjExNjY2ODQ3Mi4xNjY2OTc0NzAw*_ga_EQ6LD9QEJB*MTY2Njk3NzM0Ny4yLjEuMTY2Njk3ODQ3OS4zNi4wLjA.#operation/updateContact)
- [x] Create Newsletter | Creates a new newsletter and puts it in a queue to send. [See the docs here](https://apireference.getresponse.com/?_ga=2.47520738.499257728.1666974700-2116668472.1666974700&_gl=1*1f3h009*_ga*MjExNjY2ODQ3Mi4xNjY2OTc0NzAw*_ga_EQ6LD9QEJB*MTY2Njk3NzM0Ny4yLjEuMTY2Njk3ODQ3OS4zNi4wLjA.#operation/createNewsletter)
- [x] Find Contact | Finds a contact. [See the docs here](https://apireference.getresponse.com/?_ga=2.47520738.499257728.1666974700-2116668472.1666974700&_gl=1*1f3h009*_ga*MjExNjY2ODQ3Mi4xNjY2OTc0NzAw*_ga_EQ6LD9QEJB*MTY2Njk3NzM0Ny4yLjEuMTY2Njk3ODQ3OS4zNi4wLjA.#operation/getContactList)
- [x] Find Campaign List | Finds a campaign list. [See the docs here](https://apireference.getresponse.com/?_ga=2.250017859.499257728.1666974700-2116668472.1666974700&_gl=1*8rz7cc*_ga*MjExNjY2ODQ3Mi4xNjY2OTc0NzAw*_ga_EQ6LD9QEJB*MTY2Njk3NzM0Ny4yLjEuMTY2Njk3OTc1Mi42MC4wLjA.#operation/getCampaignList)

# Sources
- [x] New Landing Page | Emit new event when a landing page is created. [See the docs here](https://apireference.getresponse.com/?_ga=2.47520738.499257728.1666974700-2116668472.1666974700&_gl=1*1f3h009*_ga*MjExNjY2ODQ3Mi4xNjY2OTc0NzAw*_ga_EQ6LD9QEJB*MTY2Njk3NzM0Ny4yLjEuMTY2Njk3ODQ3OS4zNi4wLjA.#operation/getLandingPageList)
- [x] New Newsletter | Emit new event when a newsletter is created. [See the docs here](https://apireference.getresponse.com/?_ga=2.47520738.499257728.1666974700-2116668472.1666974700&_gl=1*1f3h009*_ga*MjExNjY2ODQ3Mi4xNjY2OTc0NzAw*_ga_EQ6LD9QEJB*MTY2Njk3NzM0Ny4yLjEuMTY2Njk3ODQ3OS4zNi4wLjA.#operation/getNewsletterList)
- [x] New Contact | Emit new event when a contact is created. [See the docs here](https://apireference.getresponse.com/?_ga=2.47520738.499257728.1666974700-2116668472.1666974700&_gl=1*1f3h009*_ga*MjExNjY2ODQ3Mi4xNjY2OTc0NzAw*_ga_EQ6LD9QEJB*MTY2Njk3NzM0Ny4yLjEuMTY2Njk3ODQ3OS4zNi4wLjA.#operation/getContactList)
- [x] New Form | Emit new event when a form is created. [See the docs here](https://apireference.getresponse.com/?_ga=2.47520738.499257728.1666974700-2116668472.1666974700&_gl=1*1f3h009*_ga*MjExNjY2ODQ3Mi4xNjY2OTc0NzAw*_ga_EQ6LD9QEJB*MTY2Njk3NzM0Ny4yLjEuMTY2Njk3ODQ3OS4zNi4wLjA.#operation/getFormList)

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4632"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible
- [ ] [Create subscription webhooks sources (vs polling sources) whenever possible](https://pipedream.com/docs/components/guidelines/#webhook-vs-polling-sources)
- [ ] [Always emit a `summary` with events](https://pipedream.com/docs/components/guidelines/#emit-a-summary)
- [ ] [Use a built-in deduping strategy wherever possible](https://pipedream.com/docs/components/guidelines/#deduping)
- [ ] When building webhook sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#webhook-sources)
- [ ] When building polling sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#polling-sources)

**Review by GTFalcao — DISMISSED** [review]
Hi @jcortes , thanks for your contribution! Looks good to me! 👏 
Moving to QA

File: components/getresponse/actions/create-newsletter/create-newsletter.mjs [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      ],
+    },
+    selectedContacts: {
+      type: "string[]",
+      label: "Selected Contacts",
+      description: "Specify which contacts you will want the message be deliver to.",
+      propDefinition: [
+        app,
+        "contactId",
+      ],
```
Comment (vunguyenhung) [inline components/getresponse/actions/create-newsletter/create-newsletter.mjs:61]:
```suggestion
      ],
      optional: true
```

File: components/getresponse/actions/create-newsletter/create-newsletter.mjs [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    },
+    selectedSegments: {
+      type: "string[]",
+      label: "Selected Search Contacts",
+      description: "Specify which search contacts you will want the message be deliver to.",
+      propDefinition: [
+        app,
+        "searchContactId",
+      ],
```
Comment (vunguyenhung) [inline components/getresponse/actions/create-newsletter/create-newsletter.mjs:70]:
```suggestion
      ],
      optional: true
```

File: components/getresponse/actions/create-newsletter/create-newsletter.mjs [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+      description: "Specify which contacts you will want the message be deliver to.",
+      propDefinition: [
+        app,
+        "contactId",
+      ],
+    },
+    selectedSegments: {
+      type: "string[]",
+      label: "Selected Search Contacts",
+      description: "Specify which search contacts you will want the message be deliver to.",
```
Comment (vunguyenhung) [inline components/getresponse/actions/create-newsletter/create-newsletter.mjs:66]:
```suggestion
      description: "Specify which segments you will want the message be deliver to.",
```

**Review by GTFalcao — APPROVED** [review]
Approved

**Pull request merged**